src/Components/Posts.js

```diff
@@ -0,0 +1,9 @@
+const Posts = () => {
+    return (
+        <p>
+           
```

Comment:
Here you would render your post components, so you would have to import it to this file. 
`
<div>
   <Post/>
   <Post/>
   <Post/>
</div>`